pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -618,6 +618,8 @@ protected void validateFunctionConfigs(FunctionConfig functionConfig) {
         protected Integer DEPRECATED_instanceIdOffset = null;
         @Parameter(names = "--instance-id-offset", description = "Start the instanceIds from this offset")
         protected Integer instanceIdOffset = 0;
+        @Parameter(names = "--runtime", description = "either THREAD or PROCESS. Only applies for Java functions")
+        protected String runtime = "THREAD";
```

Comment:
I don't think it makes sense to default to thread runtime especially when users can run functions in python which must be in process runtime.  The default should just be null.  In LocalRunner, if Java is used the default is thread runtime anyways.

Reply:
Changed